slamd/discovery/processing/algorithms/discovery_experiment.py

```diff
@@ -136,7 +136,12 @@ def preprocess_dataframe_for_output_table(self, df):
         return self.move_after_row_column(df, cols_to_move)
 
     def plot_output_space(self, df):
+        # We need to copy the entire dataframe because we will modify the uncertainty columns.
+        df = df.copy()
+        for target in self.targets:
+            df[f'Uncertainty ({target})'] /= 2
```

Comment:
If `PlotGenerator.create_target_scatter_plot()` knows about uncertainties and error bars, it should probably also know to halve uncertainties when plotting them. I would pull this into the function